README.md

```diff
@@ -124,39 +124,74 @@ a proxy tool such as [mitmproxy](https://mitmproxy.org), or even just
 a script.
 
 A particularly simple way to proxy requests to a private bitcoind node
-is to make use of the [`corsproxy`](https://www.npmjs.com/package/corsproxy)
-npm module. Instructions to install and run the module are on its
-[home page](https://www.npmjs.com/package/corsproxy).
+is to make use of [`nginx`](https://nginx.org/). Instructions to install
+and run the program are on its [download page](https://nginx.org/en/download.html).
 
-Explicitly, install `corsproxy` with
+Explicitly, install `nginx` with
 
+```bash
+# MacOS
+brew install nginx
+
+# Debian Linux
+sudo apt install nginx
 ```
-$ npm install -g corsproxy
-```
 
-and then launch corsproxy
+copy the server conifiguration file to the appropriate location
+
+```bash
+# MacOS
+mkdir -p /usr/local/etc/nginx/sites-available
+cp bitcoind.proxy /usr/local/etc/nginx/sites-available/
+ln -s /usr/local/etc/nginx/sites-available/bitcoind.proxy /usr/local/etc/nginx/servers/bitcoind
 
+# Debain Linux
+sudo mkdir -p /etc/nginx/sites-available
+sudo cp bitcoin.proxy /etc/nginx/sites-available/
+sudo ln -s /etc/nginx/sites-available/bitcoin.proxy /etc/nginx/sites-enabled/bitcoind
 ```
-$ corsproxy
+
+check that the file is copied correctly
+
+```bash
+$ nginx -t
+nginx: the configuration file /usr/local/etc/nginx/nginx.conf syntax is ok
+nginx: configuration file /usr/local/etc/nginx/nginx.conf test is successful
 ```
-You should see:
 
+Start `nginx`
+
+```bash
+# MacOS
+brew services start nginx
```

Comment:
on macOS this can prompt a system message saying nginx wants to allow incoming connections. Should include that and say to allow.

Reply:
Thanks for catching this. I added a revision to the commit.